cmake/ADIOS2ConfigCommon.cmake.in

```diff
@@ -7,6 +7,11 @@ if(ADIOS2_HAVE_C)
   find_dependency(C)
 endif()
 
+set(ADIOS2_HAVE_Fortran @ADIOS2_HAVE_Fortran@)
+if(ADIOS2_HAVE_Fortran)
+  find_dependency(Fortran)
```

Comment:
Remove the `if()..find_dependency()...else()`.  We don't want to require fortran for C++ users.